spec/features/users_can_enter_details_spec.rb

```diff
@@ -0,0 +1,56 @@
+feature 'sign up page' do
+  scenario 'user can enter email and password to sign up' do
+    visit '/users/sign_up'
+    fill_in 'Email', with: 'test@email.com'
+    fill_in 'Password', with: 'testpass'
+    fill_in 'Password confirmation', with: 'testpass'
+    click_button 'Sign up'
+    expect(page).to have_content('Welcome! You have signed up successfully.')
+  end
+
+  scenario 'cannot sign up with invalid email' do
+    Capybara.current_driver = :selenium
```

Comment:
This is why it opens up in Firefox. Could change it:
`selenium headless`
This might work, might need to be looked up.